app/models/comment.rb

```diff
@@ -2,5 +2,9 @@ class Comment < ApplicationRecord
   belongs_to :author, class_name: 'User', foreign_key: :author_id
   belongs_to :post
 
+  def update_comments_counter
+    post.increment!(:commentsCounter)
+  end
```

Comment:
- [ ] **[OPTIONAL]** I would recommend you add a callback for this method based on previous suggestions.